common/predictive-text/unit_tests/headless/worker-predict-dummy.js

```diff
@@ -0,0 +1,209 @@
+/*
+ * Unit tests for the Dummy prediction model.
+ */
+
+var assert = require('chai').assert;
+
+var DummyModel = require('../../build/intermediate').models.DummyModel;
+
+describe('LMLayerWorker dummy model', function() {
+  describe('instantiation', function () {
+    it('can be instantiated with capabilities', function () {
+      var model = new DummyModel(defaultCapabilities);
+      assert.isObject(model);
+    });
+
+    it('supports dependency-injected configuration', function () {
+      let configuration = {
+        leftContextCodeUnits: 64,
+        rightContextCodeUnits: 0
+      };
+
+      var model = new DummyModel(
+        {
```

Comment:
Minor code-styling nitpick - opening brackets should always be on the same line as their statement.

i.e.:  `var model = new DummyModel(`**`{`**